Update process.md (#102)
Quaver/Quaver.Wiki: en-US/ranking/process.md

If your mapset is denied, you will receive a comment from one or more Ranking Supervisors with the reason(s) why. After this period, you are eligible to update your map with fixes to submit again for rank. **Keep in mind that if denied, your set will be placed in a cooldown period for 3 days until you are able to submit it again.**

### 5b. Mapset On-Hold

> This is used to put mapset which have minor issues that prevent them from being ranked, such as improper metadata or overlapping notes, into a state that resets and restricts all supervisors until the mapset is resolved. 

Reviewer: ```suggestion
If your mapset is put on-hold, it means that minor issues were found that prevent it from being ranked. The most common issues are improper metadata or overlapping notes. During this state, any existing supervisor votes will be reset back to zero, and your set cannot be voted for until all issues are resolved. You will need to update your mapset in order for it to go to a status of "Resolved."
```

### 5c. Mapset Resolved

> If the status of a mapset is On Hold, it has to be updated for it to be marked as Resolved. In this state supervisors can review it again.

Reviewer: ```suggestion
If the status of your mapset is on-hold, it has to be updated for it to automatically be marked as resolved. While in this state, the supervisors can review and vote for it again. 
```

If your mapset is denied, you will receive a comment from one or more Ranking Supervisors with the reason(s) why. After this period, you are eligible to update your map with fixes to submit again for rank. **Keep in mind that if denied, your set will be placed in a cooldown period for 3 days until you are able to submit it again.**

### 5b. Mapset On-Hold

> When a mapset is put into this state, its position in the queue is retained but is awaiting further minor changes that needs to be resolved. This is used to put mapsets that are of otherwise rankable quality that is lacking in formalities which would otherwise be grounds for denial, such as improper metadata or overlapping notes, into a state where all supervisor votes are reset and bars the mapset from any further voting by supervisors, as well as keeping its queue position. 

Author: ```suggestion
This is used to put mapsets that are of otherwise rankable quality that is lacking in formalities which would otherwise be grounds for denial, such as improper metadata or overlapping notes, into a state where all supervisor votes are reset and bars the mapset from any further voting by supervisors, as well as keeping its queue position. 
```

> When a mapset is put into this state, its position in the queue is retained but is awaiting further minor changes that needs to be resolved. This is used to put mapsets that are of otherwise rankable quality that is lacking in formalities which would otherwise be grounds for denial, such as improper metadata or overlapping notes, into a state where all supervisor votes are reset and bars the mapset from any further voting by supervisors, as well as keeping its queue position. 

Author: ```suggestion
This is used to put mapsets which has minor issues that prevents it from being ranked, such as improper metadata or overlapping notes, into a state that keeps its queue position but resets and restricts all supervisor votes.
```

### 5c. Mapset Resolved

> When the creator of a mapset that has been put onto the On-Hold state updates said mapset, its status will become Resolved, meaning that it can be voted for again by any supervisor.

Author: ```suggestion
If the status of a mapset is On Hold, it has to be updated for it to be marked as Resolved. In this state supervisors can review it again
```
